iOS/DuckDuckGo/Refactoring/Updated/UpdatedOmniBarView.swift

```diff
@@ -41,7 +41,8 @@ final class UpdatedOmniBarView: UIView, OmniBarView {
     var refreshButton: UIButton! { searchAreaView.reloadButton }
     var privacyIconView: UIView? { privacyInfoContainer.privacyIcon }
     var searchContainer: UIView! { searchAreaContainerView }
-    let expectedHeight: CGFloat = Metrics.height
+    let expectedHeight: CGFloat = UpdatedOmniBarView.expectedHeight
```

Comment:
This is fine but shadowing with an static seems ... a little pointless, why not just hardcode the height right here and then use this property where it's needed, like in https://github.com/duckduckgo/apple-browsers/pull/580/files#diff-925cc5f3cf68be43e7077937522b46f39f9b247d24787dcc50604e4cf751f2a9R121 

I don't feel too strongly though, so up to you.